homeassistant/components/matter/manifest.json

```diff
@@ -6,6 +6,6 @@
   "dependencies": ["websocket_api"],
   "documentation": "https://www.home-assistant.io/integrations/matter",
   "iot_class": "local_push",
-  "requirements": ["python-matter-server==6.3.0"],
+  "requirements": ["python-matter-server==6.5.0b0"],
```

Comment:
can we mention the dep bump in the PR description please ?

Reply:
I've just added it to the list.